src/app/reserve_queue/reserve_queue.cpp

```diff
@@ -0,0 +1,95 @@
+#include "reserve_queue.hpp"
+#include <algorithm>
+
+namespace
+{
+bool addPatientIfNotPresent(ReserveQueue::PatientQueueType& patients_queued, const std::shared_ptr<Patient>& patient)
+{
+    auto patient_iterator = std::ranges::find(patients_queued, patient);
+
+    if (patient_iterator == patients_queued.end())
+    {
+        patients_queued.emplace_back(patient);
+
+        return true;
+    }
+
+    return false;
+}
+
+void removePatientFromQueue(ReserveQueue::PatientQueueType& patients_queued, const std::shared_ptr<Patient>& patient)
+{
+    if (auto patientSearched = std::ranges::find(patients_queued, patient); patientSearched != patients_queued.end())
+    {
+        patients_queued.erase(patientSearched);
+    }
+}
+} // namespace
+
+bool ReserveQueue::addAwaitingPatient(Treatment treatment, const std::shared_ptr<Patient>& patient)
```

Comment:
avoid verbs, please switch to nouns 
`addPatientToQueue `

Reply:
Function renamed.
DONE.